charts/datadog/templates/_container-agent.yaml

```diff
@@ -222,6 +247,14 @@
     - name: NVIDIA_VISIBLE_DEVICES
       value: all
     {{- end }}
+    {{- if eq  (include "should-enable-process-agent" .) "true" }}
```

Comment:
There is also a `should-enable-system-probe` macro; any reason not to use that here?

Reply:
yes, the intent is to make sure the core Agent gets the same config as the other processes. These 2 env vars are solely defined in the process Agent whether or not system-probe is enabled: https://github.com/DataDog/helm-charts/blob/d1a485fb11eb8df1faa3115489ee3ac8831683e7/charts/datadog/templates/_container-process-agent.yaml#L38-L42, so gating them behind should-enable-system-probe could lead to a discrepancy between what's on the process Agent vs what's on the core Agent